storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/Changes.java

```diff
@@ -36,7 +62,7 @@ private Changes(@NonNull Set<String> affectedTables) {
      */
     @NonNull
     public static Changes newInstance(@NonNull Set<String> affectedTables) {
-        return new Changes(affectedTables);
+        return new Changes(affectedTables, Collections.<String>emptySet());
```

Comment:
omg, this looks so weird after almost year of writing in kotlin style `Collections.emptySet<String>()`. I haven't been really writing in java for a long time now…